locales/crowdin.yml

```diff
@@ -0,0 +1,53 @@
+"project_id_env": "CROWDIN_PROJECT_ID"
+"api_token_env": "CROWDIN_TOKEN"
+"base_path": "."
+
+"files": [
+   {
+      "source": "metabase.po",
+      "translation": "%language%.po",
+      "languages_mapping": {
+         "language": {
```

Comment:
this took me a while to figure out, but we have the use the `%language%` placeholder here because it is the only one that is unique for arabic, so we can maintain support for both `ar` and `ar-SA`, so we have to create custom mappings here for all of our language codes.

(we actually might want to drop one of these arabic translations in hopes of having one high quality one, rather than splitting translation resources between two almost-identical localizations)